### DISCUSSION TITLE
Ensmble IDs to gene name

### DISCUSSION THREAD

**Discussion by ShwethaRaghunathan** [satijalab/seurat#6920] [Q&A]
Hi, 

I am very new to using Seurat and am a basic R Studio user. 

All my results - PCA, UMAP etc return their PC features or heatmaps with Ensembl IDs instead of gene names. How can I get around this? 

I've tried to use bioMart to get a list of gene names and ensemble Ids. But I am unsure how to swap this out in the Seruat object. 

Thank you for your time!

**Comment by summer-tao** [comment]
I have same problem
